apps/web/src/components/Basenames/RegistrationLearnMoreModal/index.tsx

```diff
@@ -72,7 +73,7 @@ export default function RegistrationLearnMoreModal({
               <div className="flex flex-row items-center justify-start gap-2">
                 <ImageWithLoading
                   src={coinbaseVerification as StaticImageData}
-                  alt="criteria icon"
+                  alt=""
```

Comment:
Hm, I think these should have alts, just the correct ones

Reply:
Added descriptions of the various icons.